### DISCUSSION TITLE
Paraphyletic Gleicheniales?

### DISCUSSION THREAD

**Discussion by jessiepelosi** [pteridogroup/ppg#3] [Ideas]
Hi all, 

I wasn't confident about placing this in the taxonomic proposals, but think it may be something to consider for PPG II. Recent phylogenomic studies have started to uncover that Gleicheniales may not be monophyletic as suggested by earlier work and designated by PPG I. Both nuclear (Qi et al., 2018, Pelosi et al., 2022) and plastid (Du et al., 2022) have found that Gleicheniales may be paraphyletic will possible topologies including Hymenophyllales nested within Gleicheniales or Gleicheniales forming a grade along the backbone of the ferns. One of the major drawbacks to these phylogenomic studies is that there are relatively few taxa considered and that lack of the inclusion of the family Matoniaceae, factors that may impact the topological inferences made. These studies contrast, however, with phylogenetic inferences using many more taxa but very few loci such as Testo and Sundue (2016) and Nitta et al. (2022), which find that Gleicheniales is indeed monophyletic. 

I was wondering what other folks thought about this... 

References: 
- Du, X-Y., Kuo, L-Y., Zuo, Z-Y., Li, D-Z., Lu, J-M. 2022. Structural variation of plastomes provides key insight into the deep phylogeny of ferns. Frontiers in Plant Science 13:862772. https://doi.org/10.3389/fpls.2022.862772
- Nitta, J.H., Schuettpelz, E., Ramírez-Barahona, S., and Iwasaki, W., 2022. An open and continuously updated fern tree of life. Frontiers in Plant Science. 13:909768. https://doi.org/10.3389/fpls.2022.909768 
- Pelosi, J.A., Kim, E.H., Barbazuk, W.B., and Sessa, E.B. 2022. Phylotranscriptomics illuminates the placement of whole genome duplications and gene retention in ferns. Frontiers in Plant Science. 13:882441. doi: 10.3389/fpls.2022.882441  
- Testo, W.L., and Sundue, M. 2016. A 4000-species dataset provides new insight into the evolution of ferns. Molecular Phylogenetics and Evolution. 105:200-211. 
- Qi, X., Kuo, L-Y., Gou, C., Li, H., Li, Z., Qi, J., Wang, L., Hu, Y., Xiang, J., Zhang, C., Guo, J., H., C-H., and Ma., H. 2018. A well-resolved fern nuclear phylogeny reveals the evolution history of numerous transcription factor families. Molecular Phylogenetics and Evolution 127: 961-977. https://doi.org/10.1016/j.ympev.2018.06.043

**Comment by joelnitta** [comment]
Thanks @jessiepelosi for starting this discussion.

The tree of Nitta et al. 2022 ("FTOL") has been updated since the original publication with new sequences from GenBank (and will continue to be updated). The newest tree (v1.4.0) including plastomes of *Stromatopteris*, *Sticherus*, *Dicranopteris*, *Diplopterygium*, *Matonia*, *Dipteris*, and *Cheiropleuria* shows a non-monophyletic Glecheniales.

Here is a family-level tree extracted from the full tree (all families are monophyletic):

![family_tree](https://github.com/pteridogroup/ppg/assets/13459362/25bd16ad-8ea3-4dc1-8207-55191f225344)

Here is a [link to the FTOL tree viewer zoomed in to *Matonia*](https://taxonium.org/?treeUrl=https%3A%2F%2Fraw.githubusercontent.com%2Ffernphy%2Fftol_vis%2Fmain%2F_targets%2Fuser%2Ftaxonium%2Fftol_tree.tree&ladderizeTree=true&treeType=nwk&metaUrl=https%3A%2F%2Fraw.githubusercontent.com%2Ffernphy%2Fftol_vis%2Fmain%2F_targets%2Fuser%2Ftaxonium%2Fftol_data.csv&metaType=meta_csv&xType=x_dist&color=%7B%22field%22%3A%22meta_family%22%7D&config=%7B%22source%22%3A%22GenBank%20release%20253%22%2C%22title%22%3A%22FTOL%20v1.4.0%22%7D&srch=%5B%7B%22key%22%3A%22aa1%22%2C%22type%22%3A%22name%22%2C%22method%22%3A%22text_match%22%2C%22text%22%3A%22Matonia%22%2C%22gene%22%3A%22S%22%2C%22position%22%3A484%2C%22new_residue%22%3A%22any%22%2C%22min_tips%22%3A0%7D%5D&zoomToSearch=0).

You can hover the mouse over the branches to see support values. The monophyly of *Matonia* + *Phanerosorus* + *Dipteris* + *Cheiropleuria* separate from other Gleicheniales seems well-supported.

The next problem is to find a validly published treatment for this change. Shu et al. (2022) say they "propose to redefine a general ordinal rank of Matoniales from Dipteridales Doweld (2001) and Matoniales Reveal (1993) based on the International Code of Nomenclature for algae, fungi, and plants (Shenzhen Code)", but I don't see an actual taxonomic treatment in their paper.

Shu, J.-P.; Wang, H.; Shen, H.; Wang, R.-J.; Fu, Q.; Wang, Y.-D.; Jiao, Y.-N.; Yan, Y.-H. Phylogenomic Analysis Reconstructed the Order Matoniales from Paleopolyploidy Veil. Plants 2022, 11, 1529. https://doi.org/10.3390/plants11121529 

Doweld, A. Prosyllabus Tracheophytorum: Tentamen Systematis Plantarum Vascularium (Tracheophyta); Geos: Moscow, Russia, 2001. [[Google Scholar](https://scholar.google.com/scholar_lookup?title=Prosyllabus+Tracheophytorum:+Tentamen+Systematis+Plantarum+Vascularium+(Tracheophyta)&author=Doweld,+A.&publication_year=2001)]

Reveal, J. New ordinal names for extant vascular plants. Phytologia 1993, 74, 173–177. [PDF](https://ia902501.us.archive.org/9/items/biostor-62990/biostor-62990.pdf)

(last two references copied from Shu et al. 2022. I am unable to find the original for Doweld 2001 yet).

**Comment by crothfels** [comment]
If I understand correctly, both Dipteridales and Matoniales are available, and since priority doesn't apply, one would get to pick?

For Dipteridales: http://www.plantsystematics.org/reveal/pbio/fam/famDA-DZ.html
For Matoniales: http://www.plantsystematics.org/reveal/pbio/fam/famMA-MZ.html

I think this would be a good proposal to make (the recognition of Matoniaceae+Dipteridaceae at the rank of order), if someone was willing to write it up.

**Reply by joelnitta** [reply]
@crothfels, you seem to be assuming that the solution here is to combine Dipteridales and Matoniales into a single, monophyletic order. Of course that is fine, but there is another option: to keep Dipteridales and Matoniales separate, each as their own order. It would be good if whoever makes this proposal can provide their reasoning for choosing one option vs. the other.

（also, the references in the plantsystematics.org links are the same as the ones cited in my reply above)

**Reply by jessiepelosi** [reply]
Okay, coming back to this now. I am leaning toward agreeing with @joelnitta that both Dipteridales and Matoniales should be recognized as separate orders given that they diverged in the Triassic (earliest fossils for both appear in this time), even though there are very few species in each order. There's a very nice paper (Choo and Escapa, 2018) that also shows the morphological evolution/variation in these clades that make them distinct from one another. Taken together with nuclear phylogenomics and structural plastome differences (Du et al., 2022), I'd be tempted to put them in different orders. 

I would be open to writing the proposal, but will likely need some assistance in doing so. Are we able to use "Dipteridales" and "Matoniales"? Are they valid names? From @crothfels post it looks like they are, but wanted to make sure: 

Dipteridales Doweld, Tent. Syst. Pl. Vasc.: x. 23 Dec 2001. Validated by a description in Latin. – T: Dipteris Reinw. (1825).
Matoniales Pic. Serm. ex Reveal in Phytologia 74: 175. 25 Mar 1993. Validated by a full and direct reference to Matoniaceae C. Presl, Gefässübundel Farrn: 32. 1847, associated with a description in Latin. – T: Matonia R. Br. ex Wall. (1829).

<b>References</b>

- Choo, T.Y.S, and I.H. Escapa. 2018. Assessing the evolutionary history of the fern family Dipteridaceae (Gleicheniales) by incorporating both extant and extinct members in a combined phylogenetic study. Am J Bot 105(8):1-14. https://bsapubs.onlinelibrary.wiley.com/doi/10.1002/ajb2.1121 

- Du, X-Y., L-Y. Kuo, Z-Y. Zuo, D-Z. Li, and J-M. Lu. 2022. Structural variation of playtimes provides key insight into the deep phylogeny of ferns. Front Plant Sci 13:862772. doi: 10.3389/fpls.2022.862772

**Reply by GuguYanez** [reply]
Hi! I started working with Dr. Escapa and Dr. Choo some time ago to provide morphological evidence from fossil and current taxa that helps clarify the phylogenetic proposals thrown up by new molecular data. We have found that there are basal taxa in Dipteridaceae that have plesiomorphic characters shared with Matoniaceae (Yañez et al. 2023). However, we think that fossils still have a lot of information to provide :)
In this context, I think it is a good option to consider the two families as separate orders (Dipteridales and Matoniales). I am available if you need help with the writing of the proposal Jessie.

Reference

Yañez, A., Escapa, I. H., & Choo, T. Y. (2023). Fertile Goeppertella from the Jurassic of Patagonia: mosaic evolution in the Dipteridaceae-Matoniaceae lineage. AoB Plants, 15(4), plad007.

**Comment by crothfels** [comment]
Interesting discussion, and on the impact of fossils in particular. A quick aside: I think there are some additional studies (at least one) in the pipeline that provide additional evidence in favor of Gleicheniales sensu lat. not being monophyletic. I'm not quite sure what to do with this--perhaps @jessiepelosi @GuguYanez , let me know if this proposal gets to a more advanced state, and I can inquire about that project to see if it might be published in time to support the proposal? And I realize my initial comment did imply that if we chose not to recognize a broad Gleicheniales that we'd recognize a Matoniaceae+Dipteridaceae order, but of course, that need not be so, and I personally don't have strong feelings either way. But as @joelnitta points out, it would be helpful if the proposal clearly laid out the arguments for the specific approach proposed, and ideally the "we should split gleicheniales" folks would coordinate (in this discussion?) so that we avoid having similar but mutually contradictory proposals.
